src/MonteCarlo/ParallelTempering.jl

```diff
@@ -75,6 +72,8 @@ function replica_exchange!(PT::ParallelTempering, exch_start::Int64)
     end
 end
 
+function sample! end
```

Comment:
What is the purpose of this empty function?

Reply:
Oh, I can get rid of that when I make the next commit to rename things.

Reply:
Something similar happens with the `step!` function -- in that context, the purpose is to create a place to store the doc string. Maybe do the same here?